src/libraries/icubmod/cfw2Can/cfw2can.ini

```diff
@@ -1,4 +0,0 @@
-[plugin cfw2can]
-type device
-name cfw2can
-library cfw2can
```

Comment:
Please check if `EXTRA_CONFIG WRAPPER=""` is generating a line `wrapper` in the `.ini` file.
I'd remove the `EXTRA_CONFIG` line from CMakeLists.txt

Reply:
indeed, it generates a line:
`wrapper ""`

Reply:
fixed (removed)